### DISCUSSION TITLE
Speech to text for inputing data

### DISCUSSION THREAD

**Discussion by DMMeleedy** [LostRuins/koboldcpp#261] [Ideas]
Would it be possible to add a microphone button in, where you speak and then it is converted into the input text.   Maybe you could integrate in "whisper" which is here:

https://github.com/openai/whisper

This'd be great for my roleplaying.   Thanks for considering.

**Comment by TronMetatron** [comment]
use this https://chrome.google.com/webstore/detail/voice-in-speech-to-text-d/pjnefijmagpdjfhhkpljicbbpicelgko
